pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.md.erb

```diff
@@ -10,8 +10,27 @@ theme: responsive
   details summary {
     cursor: pointer;
   }
+
+  a.tertiary-link {
+    color: #00adbc;
+    font-size: 14px;
+    font-family: "Gotham 4r", sans-serif;
+    font-weight: bold;
+    text-decoration: none;
+  }
+    a.tertiary-link .fa {
+      font-size: 12px;
+      font-weight: bold;
+      margin-left: 8px;
+    }
+
+  .optimizely-variant-a {
+    display: none;
+  }
 </style>
 
+[optimizely-original]
```

Comment:
Using the bracket-class syntax to avoid interfering with contained markdown.

Reply:
there is almost no shared content between the two versions of these pages. if there's a way to have these be two separate files, that'd probably make it easier to read.  but ok not to block deploy on this since its needed for monday. would just be nice to identify a best practice

Reply:
(e.g. via partials if those are available in this context)

Reply:
or separate files, as you mentioned as an option in the PR description

Reply:
I definitely prefer the redirect option (in fact it was my first implementation) but this experiment was specifically spec'd to show only one URL, for concern that teachers would share the experiment one directly.

I'll explore the partials option further, but it was somewhat spoiled by the bit at the end where I have to interleave some of the content to avoid including the mini contact form view more than once.